integration-tests/api/__tests__/store/__snapshots__/product-variants.js.snap

```diff
@@ -185,7 +185,7 @@ Object {
           "metadata": null,
           "option_id": Any<String>,
           "updated_at": Any<String>,
-          "value": "Handcrafted",
+          "value": "Practical",
```

Comment:
question: Where are all those strigs coming from? :D

Reply:
Honestly no idea 🤣

Reply:
In that case are we sure they will not change again? Like random value? 😅

Reply:
I've run the tests dozens of times now, and it hasn't changed sine.